integration/spark/src/main/scala/org/apache/spark/sql/test/SparkTestQueryExecutor.scala

```diff
@@ -82,6 +83,15 @@ object SparkTestQueryExecutor {
       copyResourcesifNotExists(hdfsUrl, s"$integrationPath/spark/src/test/resources",
         s"$integrationPath//spark-common-cluster-test/src/test/resources/testdatafileslist.txt")
   }
+  if (System.getProperty("useIndexServer") != null || System.getenv("useIndexServer") != null) {
```

Comment:
Please check and keep single check to useIndexServer

Reply:
ok